security-analysis/security-analysis-api/src/test/resources/SecurityAnalysisResult.json

```diff
@@ -140,5 +140,26 @@
     "branchResults" : [ ],
```

Comment:
in 1.2 we could replace by:
    "networkResult" : {
      "branchResults" : [ ],
      "busResults" : [ ],
      "threeWindingsTransformerResults" : [ ]
    }

to be consistent with the one of operatorStrategyResults